bookdown/03-pipelines.Rmd

```diff
@@ -452,11 +460,12 @@ poi = PipeOpImpute$new(param_vals =
   list(method_num = "hist", method_fct = "sample", add_dummy = "all"))
 ```
 
-we can then add a learner to the Pipeop to obtain a learner equipped with automatic
-imputation.
+A learner can thus be equipped with automatic imputation of missing values 
```

Comment:
This is an example sentence where the sentence is split into two lines? Most likely by sublime?

Reply:
dont know. The replacement does not have this?

Reply:
What is "the replacement"? In the diff the new change breaks the sentence into two sentences, l. 456 and l.457

Reply:
I think I got it now